src/components/Currency.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import { get } from 'svelte/store';
+	import { formatPlanckToDot } from '../utils/polkadot';
+	import { blockChainMeta } from '../stores';
+
+	export let value: bigint;
+</script>
+
+{formatPlanckToDot(value)}
```

Comment:
Maybe we want to change the name of this function and anything related to Dot and Planck? Maybe in another PR.